features/step_definitions/scheduler.rb

```diff
@@ -26,3 +26,25 @@
     raise "Cannot restore scheduler: #{name}" unless @result[:success]
   }
 end
+
+Given /^the #{QUOTED} clusterrole is restored after scenario$/ do |name|
+  ensure_admin_tagged
+  ensure_destructive_tagged
+  patch_json = '[{"op": "remove", "path": "/rules/2/resourceNames/1"}]'
```

Comment:
This appears to be a very scenario specific way to restore this object.
How about using `hidden recreate cluster resource( after scenario)?` step? Pls check how is it already used.